tanner/emulators/cmd_exec.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import docker
+import logging
+
+from tanner.config import TannerConfig
+from tanner.utils import patterns
+
+class CmdExecEmulator:
+	def __init__(self):
+		self.docker_client = docker.from_env(version='auto')
```

Comment:
When docker is not installed, we got an exception which freezes all the connections 
```
    self.docker_client = docker.from_env(version='auto')
  File "/usr/local/lib/python3.5/dist-packages/docker/client.py", line 79, in from_env
    **kwargs_from_env(**kwargs))
  File "/usr/local/lib/python3.5/dist-packages/docker/client.py", line 36, in __init__
    self.api = APIClient(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/docker/api/client.py", line 141, in __init__
    self._version = self._retrieve_server_version()
  File "/usr/local/lib/python3.5/dist-packages/docker/api/client.py", line 168, in _retrieve_server_version
    'Error while fetching server API version: {0}'.format(e)
docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))
```
We should provide clear manual for docker setup